godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml

```diff
@@ -1,6 +1,5 @@
 controller_list:
-  - name: ""
-    action_ns: joint_trajectory_action
+  - name: 'joint_trajectory_action'
+    action_ns: ''
```

Comment:
Yes, I will let it run automatically, but I plan to have two modes: with
rviz and without rviz.

Since for these modes, the cost time of whole procedure may be different.
 running rviz should cost more time than one without rviz.

How about we keep the current way, once we find better solution, we can
optimize it.



On Wed, Mar 9, 2016 at 10:23 AM, Jorge Nicho <notifications@github.com>
wrote:

> In
> godel_robots/abb/godel_irb2400/godel_irb2400_moveit_config/config/controllers.yaml
> <https://github.com/swri-robotics/godel/pull/1#discussion_r55544269>:
>
> > @@ -1,6 +1,5 @@
> >  controller_list:
> > -  - name: ""
> > -    action_ns: joint_trajectory_action
> > +  - name: 'joint_trajectory_action'
> > +    action_ns: ''
>
> In regards to the rviz question, creating a new file for each riviz
> configuration might be very inconvenient, unfortunately I'm not sure that
> there'd be another way around it as rviz doesn't seem to handle namespaces
> the same way other ros nodes do. However, as you'll skip the user
> interaction part then rviz isn't really necessary. Instead, you'll have a
> ros node that'd automatically goes through all the steps a human user
> would, is that correct?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swri-robotics/godel/pull/1/files#r55544269>.
>